flink-libraries/flink-table/pom.xml

```diff
@@ -45,7 +45,7 @@ under the License.
 		<dependency>
 			<groupId>org.codehaus.janino</groupId>
 			<artifactId>janino</artifactId>
-			<version>2.7.5</version>
+			<version>3.0.6</version>
```

Comment:
Hi
I checked transitive dependencies in maven. I used `dependency:tree` plugin
for janino:2.7.5 is:
```
org.codehaus.janino:janino:jar:2.7.5
+- junit:junit:jar:4.7:test
+- org.apache.ant:ant-nodeps:jar:1.7.1:compile
|  \- org.apache.ant:ant:jar:1.7.1:compile
|     \- org.apache.ant:ant-launcher:jar:1.7.1:compile
\- org.codehaus.janino:commons-compiler:jar:2.7.5:compile
```
for janino:3.0.6 is:
```
org.codehaus.janino:janino:jar:3.0.6
+- junit:junit:jar:4.12:test
|  \- org.hamcrest:hamcrest-core:jar:1.3:test
+- org.codehaus.janino:commons-compiler:jar:3.0.6:compile
\- org.apache.ant:ant:jar:1.7.1:provided
   \- org.apache.ant:ant-launcher:jar:1.7.1:provided
```